### PR TITLE
Table compute Help text fix

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/table_compute/table_compute.xml [hunk 1]
```diff
@@ -1658,8 +1658,8 @@ In Galaxy we would select the following:
  * *Type of table operation* →  **Perform a Full Table Operation**
 
    * *Operation* → **Melt**
-   * *Variable IDs* → "A"
-   * *Unpivoted IDs* → "B,C"
+   * *Variable IDs* → ``A``
+   * *Unpivoted IDs* → ``B,C``
```
Comment (wm75) [inline tools/table_compute/table_compute.xml]: If the goal is to be consistent, there are many more cases in earlier examples where literal user input is formatted in bold. You would likely want to change those cases to inline literals, too.

File: tools/table_compute/table_compute.xml [hunk 2]
```diff
@@ -1739,20 +1739,14 @@ In Galaxy we would select the following:
...
-
-         chr{elem:.0f}
+     * *Replacement value* → ``chr{elem:.0f}``
 
-       Here, the placeholder ``{elem}`` lets us refer to each element's
-       current value, while the ``:.0f`` part is a format specifier that makes
-       sure numbers are printed without decimals (for a complete description of
-       the available syntax see the
-       `Python Format Specification Mini-Language <https://docs.python.org/3/library/string.html#formatspec>`_).
+       Here, the placeholder ``{elem}`` lets us refer to each element's current value, while the ``:.0f`` part is a format specifier that makes sure numbers are printed without decimals (for a complete description of the available syntax see the `Python Format Specification Mini-Language <https://docs.python.org/3/library/string.html#formatspec>`_).
```
Comment (wm75) [inline tools/table_compute/table_compute.xml]: I'm not a huge fan of long lines for which I have to scroll horizontally. So, personally, I would keep that paragraph wrapped as it is.

File: tools/table_compute/table_compute.xml (hunk 1 above)
Comment (mtekman) [reply tools/table_compute/table_compute.xml]: will add now, thanks

File: tools/table_compute/table_compute.xml (hunk 2 above)
Comment (mtekman) [reply tools/table_compute/table_compute.xml]: will change back :-)

**Pull request merged**